querybook/webapp/const/queryExecution.ts

```diff
@@ -145,9 +145,12 @@ export interface IQueryExecutionNotification {
 }
 
 export interface IQueryValidationResult {
-    line: number;
-    ch: number;
+    start_line: number;
+    start_ch: number;
+    end_line: number | null;
+    end_ch: number | null;
```

Comment:
where are `end_line` and `end_ch` used?

Reply:
They'll be used in the upcoming UI PR (since we'll need start and end coordinates of text that needs to be changed)